Version

```diff
@@ -0,0 +1,4 @@
+DOCKER_IMAGE_ORACLE_VERSION="3.0.0"
```

Comment:
Yes this Version file is intended to be automated by the new buildContainers.sh script. (need to add it to each repo). The script builds the containers updates the Version file then commits and makes a PR to update it. This is done for 2 reasons:
• Removes the need to manually bump version numbers (quite error prone)
• (most importantly) no need for the node operates to update the quickstart to get the latest version (again error prone), since the script always pulls the most recent Version file form git